commercetools-models/src/main/java/io/sphere/sdk/productdiscounts/ProductDiscountDraft.java

```diff
@@ -14,7 +14,11 @@
         factoryMethods = {@FactoryMethod(parameterNames = {}),
                           @FactoryMethod(parameterNames = {"active", "description", "name", "predicate", "sortOrder", "value"})
 }, abstractBuilderClass = true)
-public interface ProductDiscountDraft {
+public interface ProductDiscountDraft{
```

Comment:
It would be more consistent with our other interfaces if we would add `extends WithKey` here. WDYT?